src/components/sub/menu/menu-item.jsx

```diff
@@ -1,23 +1,19 @@
-import React from 'react'
-import remixiconUrl from 'remixicon/fonts/remixicon.symbol.svg'
-import '@/components/sub/menu/styles.scss'
-import '@/components/sub/menu/menu-item.scss'
+import React from "react";
+import "@/components/sub/menu/styles.scss";
+import "@/components/sub/menu/menu-item.scss";
 
-
-
-export default ({
-  icon, title, action, isActive = null,
-}) => (
+const MenuItem = ({ children, title, action, isActive = null }) => (
   <button
-    className={`menu-item${isActive && isActive() ? ' is-active' : ''}`}
+    className={`menu-item${isActive && isActive() ? " is-active" : ""}`}
     onClick={action}
     title={title}
   >
-    <svg className="remix">
+    {children}
```

Comment:
this is part of the fix. So instead of passing a string and doing string concatenation to find the SVG to use (which I don't think you were doing correctly). 

Instead we're allowing the component that is rendering  the MenuItem to pass the icon it wants to be rendered as the child.

A contrived example consider a p element
```
<p>This is a piece of text</p>
```

in this example <p> is the react element and `This is a piece of text` is the child. Its being rendered in between the P tags.


```
<MenuItem>
    <RiBold />
</MenuItem>
```

Analogously, <MenuItem> is the react element  and <RiBold /> is the child being rendered in between the MenuItem "tags"

Reply:
and RiBold is set inside of `items[x].icon` from the menu-bar.jsx component